### ISSUE TITLE
Generate and show a complete YAML install script

### ISSUE THREAD

**Issue by rluetzner** [lutris/website#671]
## What happened

Yesterday, I ran into the first install script I needed to tweak to make it work on my local machine. I was happy when I discovered that I can look at the YAML installation script online, e.g. [here](https://lutris.net/games/install/30877/view), but I quickly realized that the YAML code cannot simply be copy and pasted. The latter part is obviously stated at the top of the page, but even so I was unable to add the missing elements to make the YAML work.

That's surprising, because one can easily download the resulting JSON file and use it directly. I tried for quite a while to generate a working YAML out of the JSON export using [yq](https://github.com/mikefarah/yq), but couldn't get it to work.

Ultimately, that is how I ended up patching the script, but JSON is so much less friendly to work with compared to YAML.

## Suggestion

Generate a full YAML script from JSON, ready to copy and paste when I click "view the installation script" for a game.

## Alternative

Add a guide how to add the missing attributes to convert the excerpt into a fully working YAML install script.

**Comment by rluetzner** [comment]
Now that I've gotten my Resident Evil 2 fix, I had another look at generating a YAML from the full JSON. It's actually pretty easy, now that I've figured it out:

```bash
cat resident-evil-2-cd-sourcenext.json | jq '.results[0]|del(.content)' | yq -p json > lutris-install-script.yaml
```

This is using `jq` and `yq`, although `yq` only takes care of the YAML output, while `jq` does all the transformation.

Here's a breakdown of what the query above does.

1. Once I've decided on a specific install script on the website, the downloaded JSON will only contain a single result. So we can take the first one.
2. The result is mostly already what we want, but Lutris gets confused when we nest another (partial) YAML inside. So we get rid of the `content` key.
